packages/i18n/src/translateWithMarkup.tsx

```diff
@@ -111,7 +116,7 @@ interface TranslateWithMarkupProps {
 function TranslateWithMarkup({
   mapping = {},
   children,
-}: TranslateWithMarkupProps) {
+}: TranslateWithMarkupProps): ReactElement | null {
```

Comment:
Isn't `ReactNode` the proper return type? This should include both `null` and all the primitives as well as JSX elements.

Reply:
It's a bit of a rabbit hole with React's `renderToStaticMarkup` expecting `ReactElement` etc.

There's probably a better way to make things work, any suggestions welcome. Just `ReactNode` (or not specifying a return type) doesn't work out of the box